zeroconf/__init__.py

```diff
@@ -2179,7 +2179,7 @@ def add_multicast_member(
                 interface,
             )
             return None
-        elif _errno == errno.EINVAL:
+        elif _errno in {errno.EINVAL} | ({errno.WSAEINVAL} if hasattr(errno, 'WSAEINVAL') else set()):
```

Comment:
This will require `getattr(errno, 'WSAEINVAL')` instead of `errno.WSAEINVAL` to satisfy Mypy for now I think

Reply:
Oh that looks like a  better solution than what I came up with.